ui/admin/app/templates/scopes/scope/targets/target/add-credential-sources.hbs

```diff
@@ -14,16 +14,17 @@
   </page.header>
 
   <page.body>
-    {{#if @model.credentialLibraries}}
-      <Form::Target::AddCredentialLibraries
+    {{#if (or @model.credentialLibraries @model.credentials)}}
+      <Form::Target::AddCredentialSources
         @model={{@model.target}}
         @credentialLibraries={{@model.credentialLibraries}}
+        @credentials={{@model.credentials}}
         @submit={{route-action 'save' @model.target}}
         @cancel={{route-action 'cancel'}}
       />
     {{/if}}
 
-    {{#unless @model.credentialLibraries}}
+    {{#unless (and @model.credentialLibraries @model.credentials)}}
```

Comment:
I don't think this is quite right, if I'm understanding this correctly, we want to display the form if we have a credential library _or_ a credential.

I believe this is actually saying don't display it only if _both_ exists
i.e. `!(credentialLibrary && credential)` 

which would display this block if only one exists.

I think it would actually be 
```suggestion
    {{#unless (or @model.credentialLibraries @model.credentials)}}
```
which is `!(credentialLibrary || credential)` or `!credentialLibrary && !credential`

I actually agree with the template lint rule to only use simple `unless` now 😆

Reply:
I think we should move this logic to a js file and keep this form simple.  similar to [this](https://github.com/hashicorp/boundary-ui/blob/2d3c94b57eed313e8b16b4adabcdb8e0432651bc/ui/admin/app/components/form/target/add-credential-libraries/index.hbs#L1)

Reply:
Great idea. Moving this to the controller!